mesh/pmesh.cpp

```diff
@@ -257,6 +257,8 @@ ParMesh::ParMesh(MPI_Comm comm, Mesh &mesh, int *partitioning_,
       BuildSharedVertMapping(nsvert, vert_element, vert_global_local);
       delete vert_element;
 
+      // FIXME: the next two lines are already done above! Any reason to do them
```

Comment:
I noticed this while working on this PR -- that's why it is here. I have not investigated much further -- it looks like both calls, here and above, were added in the same commit, cc9df02f (authored by @jakubcerveny), so there maybe a reason to have them both. In any case, I see it as a low-priority thing to investigate.

Reply:
I agree with you that it looks like a mistake to have it twice. I've run tests without it and things seem to work fine, and can't see a reason why it would be needed here. Do you see a big risk to deleting these extraneous lines as part of this PR?

Reply:
Looking at the code again, I don't see a reason to have these two calls repeated here. `SetMeshGen()` only uses the `elements` array and that does not change after the first call to `SetMeshGen()` above. Also, I don't think `meshgen` is modified after it is set from the serial mesh above. So, in the end, I think we can remove the repeated two lines. Are you also okay with that?

Reply:
Agree let's remove them!

Reply:
Done in 50a37df90.